src/api/axiosAPI.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios"
```

Comment:
Why the "axios" has double quote instead single quote? I hope you configured and use the prettier in your webstorm or didn't? Because the prettier fix this automatically and it had to change this to single quoute!